pkg/template/helper.go

```diff
@@ -56,11 +56,11 @@ func executeTestCombination(v VersionTestTemplate) error {
 	return nil
 }
 
-// AddTestCases returns the test case based on the name to be used as customflag.
-func AddTestCases(names []string) ([]testCase, error) {
-	var testCases []testCase
+// TestCases returns the test case based on the name to be used as customflag.
+func TestCases(names []string) ([]testCase, error) {
+	var tcs []testCase
```

Comment:
Let's retain testCases

Reply:
its too repetitive....in the func name we alreadu have TestCase, the name of the map is testCase...so no need to a var in use only in this scope, to be named like this.

Reply:
it will be a shadow, the map already has this name